dask/utils.py

```diff
@@ -2104,3 +2104,19 @@ def get_default_shuffle_algorithm() -> str:
         return "tasks"
     except (ImportError, ValueError):
         return "disk"
+
+
+def warn(
+    message: str | Warning,
+    category: type[Warning] | None = UserWarning,
+    stacklevel: int = 1,
+    source: Any = None,
+) -> None:
+    try:
+        import distributed
+
+        distributed.warn(message, category, stacklevel, source)
+    except ImportError:
+        import warnings
+
```

Comment:
```suggestion
```
nit: I notice that `warnings` is already imported in this module, so you don't need to import it again: https://github.com/dask/dask/blob/b86786e3e1bd5be37a24d5b9743c6c043b9daffc/dask/utils.py#L12